README.md

```diff
@@ -8,9 +8,118 @@ Ballerina Constraint Library
   [![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerina-constraint.svg?label=Last%20Commit)](https://github.com/ballerina-platform/module-ballerina-constraint/commits/master)
   [![GitHub issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-standard-library/module/constraint.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Fconstraint)
 
-This library provides features to validate the values that have been assigned to Ballerina types.
-
-The Ballerina `constraint` library facilitates APIs to do validations on the Ballerina types further with the use of annotations.
+The Ballerina `constraint` library provides annotations to add constraints to Ballerina types and an API to validate the values with respect to the constraints defined in the respective types.
+
+### Constraint annotations
+
+This library provides the following annotations on Ballerina types to validate the values created with the respective types.
+
+| Ballerina Type                    | Annotation           | Supported Constraints                                                                                                  |
+|-----------------------------------|----------------------|------------------------------------------------------------------------------------------------------------------------|
+| `int`                             | `@constraint:Int`    | `minValue`, `maxValue`, `minValueExclusive`, `maxValueExclusive`, `maxDigits`, `maxIntegerDigits`, `maxFractionDigits` |
+| `float`                           | `@constraint:Float`  | `minValue`, `maxValue`, `minValueExclusive`, `maxValueExclusive`, `maxDigits`, `maxIntegerDigits`, `maxFractionDigits` |
+| `int`&#124;`float`&#124;`decimal` | `@constraint:Number` | `minValue`, `maxValue`, `minValueExclusive`, `maxValueExclusive`, `maxDigits`, `maxIntegerDigits`, `maxFractionDigits` |
```

Comment:
Here the type denote the effective type which supports these annotations. Since number constraints are allowed to all number types, I have added it as a union of Ballerina (unique) number types. The other constraints are applied to only one effective type

Reply:
I see. maybe I got distracted because I was focusing on the types rather than the annotations themselves. Ideally, the table should be structured around the annotations. One suggestion I have is to revise the table by making the annotations the primary focus in the first column. We can then include a column titled "Associated Ballerina Type(s)" to clarify that this section is about annotations, not the types themselves. This would help emphasize the main point of the section. wdyt?

Reply:
+1, will change